WcaOnRails/app/views/contacts/website.html.erb

```diff
@@ -5,6 +5,15 @@
   <p><%= t('.description_html', contact_path: contact_path) %></p>
 
   <%= simple_form_for @contact, url: contact_website_path, html: { class: 'form-horizontal' } do |f| %>
+    <% competition_input = f.input :competition_id, as: :competition_id, only_one: true, label: false, wrapper_html: { id: "target_competition" } %>
+    <% options = [
+      [t('.inquiries.competition').html_safe + competition_input, "competition_staff"],
+      [t('.inquiries.competitions_in_general'), "wct"],
+      [t('.inquiries.wca_id_or_profile'), "wrt"],
+      [t('.inquiries.media'), "wrt"],
+      [t('.inquiries.other'), "wct"],
```

Comment:
@UnsolvedCypher should WCT get all of those inquiries or should some be WRT-only? Please let me know where should each of these go =)